cypress/integration/advancedOption.test.ts

```diff
@@ -0,0 +1,71 @@
+import { PoolName } from "../../src/constants"
+
+context("Advanced option test", () => {
+  // have two seperate maps here since the naming convention is different throughout the page
+  const pools = ["BTC Pool V2", "Stablecoin Pool V2"]
+  const advancedOptionValues = ["0.1", "20", "20"]
+
+  beforeEach(() => {
+    const host = Cypress.env("DAPP_HOST") as string
+    cy.visit(`${host}#/pools`)
+    cy.waitForReact()
+    cy.wait(3000)
+  })
+
+  function advancedOptionTest(poolName: PoolName) {
+    const host = Cypress.env("DAPP_HOST") as string
+    cy.visit(`${host}#/pools`)
+    cy.wait(3000)
+    cy.contains(poolName)
+      .parents(".poolOverview")
+      .within(() => {
+        cy.get("button").contains("Withdraw").click()
+      })
+    cy.wait(10000)
+
+    cy.get("[data-testid=optionContainer]").should("exist")
```

Comment:
This is a good start. These names will work now but if there are other components with `option container` in the future then this test could require maintenance to in case it targets the other "option container", it could clash with this targeting. Something like `advOptionContainer` could be a better name just to scope it just a little more to this component.